datahub-graphql-core/README.md

```diff
@@ -8,7 +8,7 @@ permitting reads and writes against the entities and aspects on the Metadata Gra
 
 Contained within this module are 
 
-1. **GMS Schema**: A GQL schema that based on GMS models, located under `resources/gms.graphql`. 
```

Comment:
There is no folder `datahub-graphql-core/resources/` either. So what is this new location pointing to?

Reply:
`src/main/resources`

Reply:
This doc will go into public docs website. So we will need a URL that people looking at public docs can understand.

Reply:
updated to add link.